src/FlightDisplay/FlyViewMap.qml

```diff
@@ -533,13 +533,13 @@ FlightMap {
         }
 
         onClicked: {
-            if (guidedActionsController.guidedUIVisible || (!guidedActionsController.showGotoLocation && !guidedActionsController.showOrbit)) {
+            if (guidedActionsController.guidedUIVisible || (!guidedActionsController.showGotoLocation && !guidedActionsController.showOrbit && !guidedActionsController.showROI)) {
                 return
             }
             orbitMapCircle.hide()
             gotoLocationItem.hide()
             var clickCoord = _root.toCoordinate(Qt.point(mouse.x, mouse.y), false /* clipToViewPort */)
-            if (guidedActionsController.showGotoLocation && guidedActionsController.showOrbit) {
+            if (guidedActionsController.showGotoLocation && (guidedActionsController.showOrbit || guidedActionsController.showROI)) {
```

Comment:
This isn't correct. It won't do the right thing if goto is not shown but orbit and roi are. This block should be change like this:

```
if (guidedActionsController.showGotoLocation && !guidedActionsController.showOrbit && !guidedActionsController.showROI) {
    ...
}  else if (guidedActionsController.showOrbit && ! guidedActionsController.showGotoLocation && !guidedActionsController.showROI) {
    ....
} else if (...) {
    ....
} else {
    Handle the case where a menu needs to be shown to pick from
}

Reply:
of goto is enabler for ROI and Orbit then the if case is very simple
I made the changes and committed

Reply:
This is still incorrect since it doesn't follow what I showed above.